docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+### What else can we do to improve MUI for you?
+
+<img src="/static/blog/2021-survey/6.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Word cloud of what we can do to improve MUI" />
+
+<p class="blog-description">1007 out of 1589 answered this question in 2021</p>
+
+Here are the most recurring topics:
+
+- **More components:** We continue to see requests for more advanced components, following last year's pattern, such as Charts, Forms, and Calendar, to name a few.
+- **More examples:** Knowing the "officially recommended" ways of using any product is always helpful. We know that many of the learning resources you find out there right now are outdated (based on v4), so that's absolutely worth of investing.
+- **Provide more themes:** Even with the launch of Material v3, Material Design has long been a design language that some find outdated. That's why we are working on a second design system, thought to offer everything that the current Material package does (component quantity and quality) but with an alternative design direction.
+- **Fewer breaking changes:** MUI Core v5 definitely introduced some important breaking changes, especially because of the new styling solution. We’ll continue to explore the v5 migration topic across the post but, for now, it's good to share that we don’t expect to release any major this year. In fact, we aim for majors happening with at least 12 months apart from each other.
+- **Improve customization:** Many of the entries for customization ask for making it easier, to add more examples of common customization use-cases (like font-family and primary/secondary color change), and improvements to theme capabilities. Even with the popularity of emotion and styled-components, there continues to be an enourmous need to facilitate customization.
+
+If you have requests or ideas for improving the above-mentioned topics, here are a few tips to have them prioritized faster:
+
+- In both MUI Core and X repositories, you should label a request issue with `Waiting for upvotes`. If your issue gains a significant amount of upvotes, it will definitely get our attention and consideration.
+- When requesting more components, make sure to benchmark existing implementations of it in the wild as it’s very helpful to see where they succeed and fail, as upfront learning content. It’s also great to explain as best as you can what is the problem you’re trying to solve. Oftentimes, an already available component might already solve it.
```

Comment:
singular/plural misalignment. Also, I think this is really two different points: show us some benchmarks on the one hand, and clearly describe your problem on the other hand. In fact, I might rearrange this list so that the latter point is the first one, above the point about upvotes, because it negates all other points if it turns out the component they need already exists.

```suggestion
- When requesting a new component, make sure to benchmark existing implementations of it in the wild so we can see what works and what we could better. 

- Try to explain the problem you're having as clearly as possible—it's often the case that we already have a component to solve it.
```